src/materials/MeshPhysicalMaterial.js

```diff
@@ -39,6 +39,12 @@ class MeshPhysicalMaterial extends MeshStandardMaterial {
 			}
 		} );
 
+		this.iridescenceMap = null;
+		this.iridescenceIOR = 1.3;
+		this.iridescenceThicknessMinimum = 100;
+		this.iridescenceThicknessMaximum = 400;
```

Comment:
I think this is the first time we have these type of properties in the material.

Would it make sense to unify them into `iridescenceThicknessRange`? ie..

```js
this.iridescenceThicknessRange = [ 100, 400 ];
```

Reply:
@mrdoob I changed the PR to use a range instead of separated values. Also updated the editor.